community/pdns-recursor/APKBUILD

```diff
@@ -2,11 +2,10 @@
 # Maintainer: tcely <pdns-recursor+aports@tcely.33mail.com>
 pkgname=pdns-recursor
 pkgver=4.1.12
-pkgrel=1
+pkgrel=2
```

Comment:
I think no reason to bump, there's no package for s390x so it will be added but other arches could remain as is

Reply:
I'd rather bump than have different "source" versions in the same pkgver/pkgrel descriptor.